docs/content/querying/dimensionspecs.md

```diff
@@ -194,11 +200,11 @@ Example for the `__time` dimension:
 }
 ```
 
-### Lookup extraction function
+### Lookup Extraction Function
 
 Lookups are a concept in Druid where dimension values are (optionally) replaced with new values. 
-For more documentation on using lookups, please see [here](../querying/lookups.html). 
-Explicit lookups allow you to specify a set of keys and values to use when performing the extraction.
+A lookup can be of type `namespace` or `map`. A `map` lookup is passed as part of the query. 
```

Comment:
Sorry for the confusion, `namespace` one possible implementation and it is going away. Currently we are re designing how lookup works to make it production ready. please find the road map here https://github.com/druid-io/druid/issues/2329